### DISCUSSION TITLE
Any idea why this key is giving "undefined" value?

### DISCUSSION THREAD

**Discussion by bakerhaider** [swellstores/community#442] [Help] [answered]
This is my code:   ` 
 const { key } = useParams();
    console.log(key)

  const onSubmit = async () => {

    const res = await swell.account.recover({

      password: password,

      reset_key: "3647bd91cfb130e3f017be4a9b2f441a",

    })
    console.log("sent",res)
  } `

It works when i hardcode the reset_key but not when getting it via params

**Comment by logeshswell** [comment, marked as answer]
Then, it's not an issue with swell system (or sdk) but with your implementation.
Sorry, we couldn't debug such issues from our end.
